test/distributed/_composable/fsdp/test_fully_shard_comm.py

```diff
@@ -660,5 +660,22 @@ def forward(self, x: torch.Tensor):
             self.assertEqual(losses[0], losses[1])
 
 
+class TestFullyShardUnshardMultiThread(FSDPTestMultiThread):
+    @property
+    def world_size(self) -> int:
+        return 2
+
+    @unittest.skipIf(not TEST_CUDA, "no cuda")
+    def test_unshard_no_param_group(self):
+        # Check that we can call `unshard()` on a module with no parameter
+        # group / no managed parameters without erroring
+        model = nn.Sequential(nn.Linear(4, 4), nn.Linear(4, 4))
+        for lin in model:
+            fully_shard(lin)
+        fully_shard(model)
+        handle = model.unshard(async_op=True)
+        handle.wait()
```

Comment:
was the error coming from ``None.wait()`` ?

Reply:
yep!